pom.xml

```diff
@@ -393,6 +393,7 @@
                         <sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>
 <sourceDirectory>scenarios/okhttp-scenario</sourceDirectory>
 <sourceDirectory>scenarios/spring-4.3.x-scenario</sourceDirectory>
+<sourceDirectory>scenarios/nats-client-scenario</sourceDirectory>
```

Comment:
I think these lines are not necessary. The checkstyle would cover all scenarios folders in the runtime.

Reply:
ok , I remove  other  scenarios sourceDirectory  together ?

Reply:
Yes.